### PR TITLE
Add anchor tag for link column

### LINKED ISSUE #33: Website link should be hyperlinked inside the table

### PR DESCRIPTION
**Pull request by Guru107**

### CODE REVIEW

File: src/components/Category.js [hunk 1]
```diff
@@ -74,9 +74,11 @@ const CategoryComponent = ({ category, stateContext }) => {
         })
 
         let updatedColumns = columns
+        .map(i => i.key === "Link" ? {...i, render: (link) => <a href={link} target="_blank" rel="noreferrer">{link}</a>} : i )
+        
         if (!isExternalResources) {
           updatedColumns = [
-            ...columns,
+            ...updatedColumns,
```
Comment (jigar1101) [inline src/components/Category.js]: The last time I checked this was causing issue due to re-render and double verification column was being added on all pages. Can you verify if this is not happening?

File: src/components/Category.js [hunk 2]
```diff
@@ -74,9 +74,11 @@ const CategoryComponent = ({ category, stateContext }) => {
         })
 
         let updatedColumns = columns
+        .map(i => i.key === "Link" ? {...i, render: (link) => <a href={link} target="_blank" rel="noreferrer">{link}</a>} : i )
```
Comment (jigar1101) [inline src/components/Category.js]: Would suggest adding this here - https://github.com/COVID19-Resources-India/covid19resourcesindia/blob/6805d279567509a3fd0cbaf180575c5d2b5d4c00/src/constant/columns.js#L5

File: src/components/Category.js (hunk 1 above)
Comment (Guru107) [reply src/components/Category.js]: ok

Comment (Guru107) [reply src/components/Category.js]: Checked all tabs there isn't any duplication of verification columns.

File: src/components/Category.js (hunk 2 above)
Comment (Guru107) [reply src/components/Category.js]: Thought of that, but didn’t add it there as there was no UI logic in that file. Let me know, can add it there shouldn’t be an issue

Comment (jigar1101) [reply src/components/Category.js]: Yes better to add there as we are already looping over there

**Review by jigar1101 — APPROVED** [review]

**Pull request merged**